core/src/main/java/org/elasticsearch/search/aggregations/bucket/sampler/InternalSampler.java

```diff
@@ -48,11 +48,6 @@ public String getWriteableName() {
     }
 
     @Override
-    protected String getType() {
-        return "sampler";
```

Comment:
I think the idea was that InternalSampler and UnmappedSampler should have the same type, so that we only parse into InternalSampler. Maybe we meant to call it "sampler" in both cases rather than mapped_sampler but I guess we don't need that.